besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1090,6 +1093,11 @@ public void run() {
     if (network != null && network.isDeprecated()) {
       logger.warn(NetworkDeprecationMessage.generate(network));
     }
+    if (network == EPHEMERY) {
+      ephemeryGenesisFile =
+          new EphemeryGenesisFile(readGenesisConfigFile(), readGenesisConfigOptions());
+      ephemeryGenesisFile.updateGenesis();
+    }
```

Comment:
I actually think lines 1096 - 1100 should be part of readGenesisConfigFile() so that calling readGenesisConfigFile will always give you the effective genesis file, and then maybe you don't need to modify the EPHEMERY networkName object itself.

Reply:
Moving the code to this readGenesisConfigFile() resulted in a recursive error. I placed the code where it is now because I want the check to happen on client start